docs/GettingStarted.md

```diff
@@ -64,7 +64,8 @@ of the Datadog tracer, you can override the following defaults:
       tracer: Datadog.tracer,
       debug: false,
       trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126
+      trace_agent_port: 8126,
+      env: Rails.env
```

Comment:
@ufoot not sure about that; probably I may prefer a generic `tags` options so that I can:
```ruby
Rails.configuration.datadog_trace = {
      enabled: true,
      auto_instrument: true,
      auto_instrument_redis: true,
      tags: { 'env' => Rails.env, 'another' => 'with_value' }
}
```

what do you think?